drivers/bridge/setup_ip_tables.go

```diff
@@ -13,14 +13,43 @@ const (
 	DockerChain = "DOCKER"
 )
 
-func (n *bridgeNetwork) setupIPTables(config *networkConfiguration, i *bridgeInterface) error {
+func setupIPChains(config *configuration) (*iptables.ChainInfo, *iptables.ChainInfo, error) {
 	// Sanity check.
 	if config.EnableIPTables == false {
-		return IPTableCfgError(config.BridgeName)
+		return nil, nil, fmt.Errorf("Cannot create new chains, driver EnableIPTable is disabled")
```

Comment:
All other existing error messages start with upper case. I am happy to make the change to follow the convention but then I have to do it for all previously existing error messages at least in this file to remain consistent. Is that what you are asking for?